cmd/terramate/e2etests/exp_trigger_cloud_test.go

```diff
@@ -236,6 +224,158 @@ func TestCloudTriggerUnhealthy(t *testing.T) {
+					},
+					Status:           stack.OK,
+					DeploymentStatus: deployment.OK,
+					DriftStatus:      drift.Unknown,
+				},
+			},
+			flags: []string{`--experimental-status=ok`},
+			want: want{
+				trigger: runExpected{
+					StdoutRegex: "Created trigger for stack",
+				},
+				list: runExpected{
+					Stdout: nljoin("s2"),
+				},
+			},
+		},
+		{
+			name: "1 cloud stack failed, other ok, asking for healthy: trigger ok",
+			layout: []string{
+				"s:s1:id=s1",
+				"s:s2:id=s2",
+			},
+			stacks: []cloud.StackResponse{
+				{
+					ID: 1,
+					Stack: cloud.Stack{
+						MetaID:     "s1",
+						Repository: "github.com/terramate-io/terramate",
+					},
+					Status:           stack.Failed,
+					DeploymentStatus: deployment.Failed,
+					DriftStatus:      drift.Unknown,
+				},
+				{
+					ID: 2,
+					Stack: cloud.Stack{
+						MetaID:     "s2",
+						Repository: "github.com/terramate-io/terramate",
+					},
+					Status:           stack.OK,
+					DeploymentStatus: deployment.OK,
+					DriftStatus:      drift.Unknown,
+				},
+			},
+			flags: []string{`--experimental-status=ok`},
+			want: want{
+				trigger: runExpected{
+					StdoutRegex: "Created trigger for stack",
+				},
+				list: runExpected{
+					Stdout: nljoin("s2"),
+				},
+			},
+		},
+		{
+			name: "1 cloud stack failed, other ok, asking for failed: trigger failed",
+			layout: []string{
+				"s:s1:id=s1",
+				"s:s2:id=s2",
+			},
+			stacks: []cloud.StackResponse{
+				{
+					ID: 1,
+					Stack: cloud.Stack{
+						MetaID:     "s1",
+						Repository: "github.com/terramate-io/terramate",
+					},
+					Status:           stack.Failed,
+					DeploymentStatus: deployment.Failed,
+					DriftStatus:      drift.Unknown,
+				},
+				{
+					ID: 2,
+					Stack: cloud.Stack{
+						MetaID:     "s2",
+						Repository: "github.com/terramate-io/terramate",
+					},
+					Status:           stack.OK,
+					DeploymentStatus: deployment.OK,
+					DriftStatus:      drift.Unknown,
+				},
+			},
+			flags: []string{`--experimental-status=failed`},
+			want: want{
+				trigger: runExpected{
+					StdoutRegex: "Created trigger for stack",
+				},
+				list: runExpected{
+					Stdout: nljoin("s1"),
+				},
+			},
+		},
+		{
+			name: "1 cloud stack drifted, other ok, asking for unhealthy: trigger drifted",
+			layout: []string{
+				"s:s1:id=s1",
+				"s:s2:id=s2",
+			},
+			stacks: []cloud.StackResponse{
+				{
+					ID: 1,
+					Stack: cloud.Stack{
+						MetaID:     "s1",
+						Repository: "github.com/terramate-io/terramate",
+					},
+					Status:           stack.Drifted,
+					DeploymentStatus: deployment.Failed,
+					DriftStatus:      drift.Drifted,
+				},
+				{
+					ID: 2,
+					Stack: cloud.Stack{
+						MetaID:     "s2",
+						Repository: "github.com/terramate-io/terramate",
+					},
+					Status:           stack.OK,
+					DeploymentStatus: deployment.OK,
+					DriftStatus:      drift.Unknown,
+				},
+			},
+			flags: []string{`--experimental-status=drifted`},
```

Comment:
```suggestion
			flags: []string{`--experimental-status=unhealthy`},
```

Reply:
I think here I forgot to update the testcase description, the intention was testing for the `drifted` status. Will change the description. Thanks for catching it.